community/contributing/index.md

```diff
@@ -8,22 +8,34 @@ redirect_from: /docs/contributing-to-spinnaker
+## Connecting with the community
 
-We'll probably end up doing something like this, but at the moment we're exploring options that might make it easier for most of you to contribute. Please stay tuned...
+You can connect with contributors and discuss opportunities to contribute in a few ways:
 
-In the meantime, if you're really excited about contributing and don't want to wait until we have all of this worked out, please reach out to us on [Slack](http://join.spinnaker.io). We love accepting community patches!
+- Reach out to us on [Slack](http://join.spinnaker.io)
+- [Check the calendar](https://calendar.google.com/calendar/u/0/embed?showPrint=0&showCalendars=0&mode=AGENDA&height=500&wkst=1&bgcolor=%23FFFFFF&src=spinnaker.io_tr65rjf5mfij7p6vucprkhulcc@group.calendar.google.com&color=%2342104A) and attend a meeting for a [SIG (Special Interest Group)](https://github.com/spinnaker/governance/blob/master/sig-index.md) that focuses on a topic relevant to you
+- Search [Slack](http://join.spinnaker.io) for channels related to topics you're interested in, and use channels beginning with `#sig-` to engage with SIGs outside of scheduled meetings
+- Join the next [Spinnaker Gardening Days](/community/gardening/) event
 
-For more details about the patch submission process, look at [How to submit a patch](/community/contributing/submitting/).
 
-## How to run Spinnaker locally for development
+## Spinnaker development environments
 
-Running Spinnaker locally on your machine is a great option for testing changes before submitting a pull request.
+The Spinnaker community is currently working to develop and document a standard environment and toolkit for Spinnaker development. While we get organized, the [Development environments](/community/gardening/dev-environment/) guide is the most current resource and focuses on development instances of Spinnaker that leverage Kubernetes. If you prefer to set up [Spinnaker services](https://spinnaker.io/reference/architecture/) locally without using Kubernetes, follow this [guide to setting up a dev environment](/guides/developer/getting-set-up/) to get up and running.
 
-Follow the [guide to setting up a dev environment](/guides/developer/getting-set-up/) to get up and running.
+Please join the [#dev-toolkit](https://spinnakerteam.slack.com/archives/C011LUJ0UQJ) channel in [Slack](http://join.spinnaker.io) to share your feedback or help develop the toolkit.
+
+## License agreements
+
+We should certainly have a place where we tell you about Contributor License Agreements and how you need to identify the right one and sign it before you are able to contribute code to Spinnaker. We'll probably end up doing something like this, but at the moment we're exploring options that might make it easier for most of you to contribute. Please stay tuned...
+
+In the meantime, you can currently contribute according to the Apache license at your own risk.
```

Comment:
What are the risks? I think the language we had before was a little less...alarming?

Reply:
@dorbin good question - I do not know the risks, which was my motivation for writing this. The language before basically is a tacit encouragement for folks to NOT contribute and that's what I want to avoid:

```
This is normally the place where we’d tell you about Contributor License Agreements and how you need to identify the right one and sign it before you are able to contribute code to Spinnaker.

We’ll probably end up doing something like this, but at the moment we’re exploring options that might make it easier for most of you to contribute. Please stay tuned…

In the meantime, if you’re really excited about contributing and don’t want to wait until we have all of this worked out, please reach out to us on Slack. We love accepting community patches!
```

I don't like that this leaves it as....we don't have this figured out so you should wait for us to do that or else bug us on Slack. Doesn't seem scalable or particularly professional. But I agree that my wording is alarming. Do you have any suggestions? I got started on a project to look into the licensing stuff so we could give people real information but it wasn't anyone else's priority and I don't have the bandwidth. Should we just not talk about licensing altogether?

Reply:
@plumpy, what say ye?